spec/controllers/saml_idp_controller_spec.rb

```diff
@@ -829,10 +829,10 @@ def stub_requested_attributes
       expect(subject).to have_actions(
         :before,
         :disable_caching,
-        [:validate_saml_request, only: :auth],
-        [:validate_service_provider_and_authn_context, only: :auth],
-        [:store_saml_request, only: :auth],
-        [:add_sp_metadata_to_session, only: :auth]
+        :validate_saml_request,
+        :validate_service_provider_and_authn_context,
+        :store_saml_request,
+        :add_sp_metadata_to_session
```

Comment:
If you mean what does this test do, it tests to see if this particular controller specifies these specific callbacks.